python/paddle/dataset/conll05.py

```diff
@@ -87,12 +87,12 @@ def reader():
             sentences = []
             labels = []
             one_seg = []
-            for word, label in itertools.izip(words_file, props_file):
+            for word, label in zip(words_file, props_file):
                 word = word.strip()
                 label = label.strip().split()
 
                 if len(label) == 0:  # end of sentence
-                    for i in xrange(len(one_seg[0])):
```

Comment:
maybe we should import six.moves

Reply:
I will change xrange to ```six.moves``` in next commit

Reply:
Actually, xrange and range does the samething.. so maybe ```range``` is just fine for our work